### DISCUSSION TITLE
GH Project not displayed in issue anymore

### DISCUSSION THREAD

**Discussion by kiview** [community/community#39747] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Today we came around the situation, that we can still add an issue to a project from within the Issue view, but the project is never shown in the issue after selection and therefore fields also cannot be edited from within the issue view. 

Selecting the project:
![image](https://user-images.githubusercontent.com/5088104/203312424-7524d6c1-0a31-4da4-a080-3f07ffe3dc92.png)

Selected project not visible:
![image](https://user-images.githubusercontent.com/5088104/203312486-0b209ed8-82ee-4639-8aaa-c904f15d2aca.png)

We did not make any change to our project; we discovered this behavior today.  However, I just discovered that this issue only happens with our public repos, private repos work as expected (the project is private as well).

**Comment by lsh-0** [comment]
I'm seeing this as well. Further, when de-selecting and then selecting the project again from the drop down, I get this:

![Screenshot at 2022-11-23 10-13-58](https://user-images.githubusercontent.com/2142748/203443355-0fa6d944-8377-4f68-8060-14d48e6b8d7a.png)

**Reply by dongsupark** [reply]
I am seeing it as well.

**Comment by mzur** [comment, marked as answer]
It seems to be working again. Can anyone confirm this?

**Reply by lsh-0** [reply]
I can confirm

**Reply by kiview** [reply]
Works for me as well, I just mark your answer to indicate this is fixed @mzur 👍
